### PR TITLE
*: set stats lease.

### PR DESCRIPTION
**Pull request by hanfei1991**
distinguish stats lease with ddl lease

### CODE REVIEW

File: tidb-server/main.go [hunk 1]
```diff
@@ -51,7 +51,8 @@ var (
 	host            = flag.String("host", "0.0.0.0", "tidb server host")
 	port            = flag.String("P", "4000", "tidb server port")
 	statusPort      = flag.String("status", "10080", "tidb server status port")
-	lease           = flag.String("lease", "1s", "schema lease duration, very dangerous to change only if you know what you do")
+	ddlLease        = flag.String("lease", "1s", "schema lease duration, very dangerous to change only if you know what you do")
+	statsLease      = flag.String("stasLease", "1s", "stats lease duration, very dangerous to change only if you know what you do")
```
Comment (shenli) [inline tidb-server/main.go]: How to choose the statsLease?

Comment (hanfei1991) [inline tidb-server/main.go]: as the user like

File: domain/domain.go [hunk 2]
```diff
@@ -471,19 +472,19 @@ func (do *Domain) StatsHandle() *statistics.Handle {
 
 // CreateStatsHandle is used only for test.
 func (do *Domain) CreateStatsHandle(ctx context.Context) {
-	do.statsHandle = statistics.NewHandle(ctx)
+	do.statsHandle = statistics.NewHandle(ctx, 0)
```
Comment (shenli) [inline domain/domain.go]: Why not use do. statsLease?

File: statistics/statscache.go [hunk 3]
```diff
@@ -45,6 +46,8 @@ type Handle struct {
 	listHead *SessionStatsCollector
 	// We collect the delta map and merge them with globalMap.
 	globalMap tableDeltaMap
+
+	Lease time.Duration
```
Comment (shenli) [inline statistics/statscache.go]: Add comment.

File: domain/domain.go [hunk 4]
```diff
@@ -374,12 +375,12 @@ type etcdBackend interface {
 }
 
 // NewDomain creates a new domain. Should not create multiple domains for the same store.
-func NewDomain(store kv.Storage, lease time.Duration, factory pools.Factory) (d *Domain, err error) {
+func NewDomain(store kv.Storage, ddlLease time.Duration, statsLease time.Duration, factory pools.Factory) (d *Domain, err error) {
```
Comment (coocood) [inline domain/domain.go]: The `statsLease` is not used.

File: tidb-server/main.go (hunk 1 above)
Comment (coocood) [inline tidb-server/main.go]: The stats lease is not as dangerous as ddl lease.
And the default 1s is too small.

**Pull request merged**